### DISCUSSION TITLE
정규화를 통해 중복 데이터를 제거하면 생기는 이점은?

### DISCUSSION THREAD

**Discussion by donghyun0218** [8percent/rdb-study#34] [Q&A]
중복 제거를 하면 좋다! 라고는 이야기를 많이 들었으나, 구체적으로 어떤 이유로 인해 좋을까요?

**Comment by wer153** [comment]
[IT위키](https://itwiki.kr/w/%EB%8D%B0%EC%9D%B4%ED%84%B0%EB%B2%A0%EC%9D%B4%EC%8A%A4_%EC%A0%95%EA%B7%9C%ED%99%94)

**Comment by donghyun0218** [comment]
1정규화: 필드 원자성
2정규화: 
3정규화: A = B, B  = C 일 때, A = C 가 성립하는 구조 정리
